webextensions/api/scripting.json

```diff
@@ -0,0 +1,393 @@
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari_ios": {
+                "version_added": null,
+                "notes": "Available for use in Manifest V3 only."
+              }
+            }
+          }
+        },
+        "executeScript": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/executeScript",
+            "support": {
+              "chrome": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "edge": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox_android": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "opera": {
+                "version_added": "74",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari": {
+                "version_added": "15.4",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari_ios": {
+                "version_added": "15.4",
+                "notes": "Available for use in Manifest V3 only."
+              }
+            }
+          }
+        },
+        "getRegisteredContentScripts": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/getRegisteredContentScripts",
+            "support": {
+              "chrome": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "edge": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox_android": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "opera": {
+                "version_added": "74",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari": {
+                "version_added": null,
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari_ios": {
+                "version_added": null,
+                "notes": "Available for use in Manifest V3 only."
+              }
+            }
+          }
+        },
+        "insertCSS": {
```

Comment:
Need specific note for properties (in `insertCSS` and `removeCSS`):
- `origin` not supported in Safari.

Reply:
Updated